orchestration/dagster_orchestration/hca_orchestration/contrib/bigquery.py

```diff
@@ -0,0 +1,65 @@
+from google.cloud import bigquery
+from google.cloud.bigquery import ExternalConfig, WriteDisposition
+
+
+def build_query_job_using_external_schema(
+        query: str,
+        source_paths: list[str],
+        schema: list[dict[str, str]],
+        table_name: str,
+        destination: str,
+        bigquery_client: bigquery.client.Client,
+        bigquery_project: str
+) -> bigquery.QueryJob:
+    """
+    Performs a bigquery query using an external table definition, using the supplied
+    schema and source paths. Results are deposited at the supplied GCS destination
+    """
+    job_config = bigquery.QueryJobConfig()
```

Comment:
idle q: any reason to namespace these rather than reference them directly?

Reply:
Juggling between the bigquery.client and storage.client types I believe.